### PR TITLE
WFLY-10999 Upgrade Infinispan to 9.4.0.Final

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.jboss.org/browse/WFLY-10999

### CODE REVIEW

**Comment by kabir** [comment]
@tommaso-borgato Are you able to pre-check this one?

**Comment by tommaso-borgato** [comment]
@kabir I have run some tests and everything looks OK

File: pom.xml [hunk 1]
```diff
@@ -240,7 +240,7 @@
         <version.io.opentracing.jaxrs2>0.1.7</version.io.opentracing.jaxrs2>
         <version.io.opentracing.tracerresolver>0.1.5</version.io.opentracing.tracerresolver>
         <version.io.opentracing.servlet>0.1.0</version.io.opentracing.servlet>
-        <version.io.reactivex.rxjava>2.1.16</version.io.reactivex.rxjava>
+        <version.io.reactivex.rxjava>2.2.2</version.io.reactivex.rxjava>
```
Comment (bstansberry) [inline pom.xml]: @asoldano Please approve, since it looks like Resteasy uses this.

**Review by asoldano — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (asoldano) [reply pom.xml]: @bstansberry approved, we're currently testing with version 2.2.2 on RESTEasy master

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
retest this please

**Pull request merged**